sdk/keyvault/microsoft-azure-keyvault-cryptography/pom.xml

```diff
@@ -21,10 +21,25 @@
 	<url>https://github.com/Azure/azure-sdk-for-java</url>
 
     <distributionManagement>
-        <site>
-            <id>azure-java-build-docs</id>
-            <url>${site.url}/site/${project.artifactId}</url>
-        </site>
+      <site>
+          <id>azure-java-build-docs</id>
+          <url>${site.url}/site/${project.artifactId}</url>
+      </site>
+      <relocation>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-security-keyvault-keys</artifactId>
```

Comment:
Can relocation section have multiple artifacts? We can include keys, secrets and certificates if that's possible.

Reply:
I checked the pom definition, it doesn't look defined.